reposerver/repository/repository.go

```diff
@@ -1236,16 +1237,26 @@ func getResolvedValueFiles(
 		}
 
 		if !isRemote {
-			_, err = os.Stat(string(resolvedPath))
-			if os.IsNotExist(err) {
+			paths, err := filepath.Glob(string(resolvedPath))
```

Comment:
filepath.Glob is pretty limited and doesn't support, e.g. double-star patterns: https://pkg.go.dev/path/filepath#Glob

I'd recommend using this, since Argo CD already uses it in several places: https://github.com/bmatcuk/doublestar

Reply:
Thanks !
I switched to doublestar with the latest commit